app/containers/Applications/ApplicationDetailsContainer.tsx

```diff
@@ -13,71 +15,228 @@ import ApplicationDetailsCardItem from './ApplicationDetailsCardItem';
 interface Props {
   query: ApplicationDetailsContainer_query;
   application: ApplicationDetailsContainer_application;
-  relay: RelayProp;
+  relay: RelayRefetchProp;
   review: boolean;
 }
 
 export const ApplicationDetailsComponent: React.FunctionComponent<Props> = props => {
   const formResults = props.application.orderedFormResults.edges;
-  const previousFormResults = props.review
-    ? props?.application?.previousSubmittedRevision
-        ?.formResultsByApplicationIdAndVersionNumber?.edges
+  const diffFromResults = props.review
+    ? props?.query?.old?.orderedFormResults?.edges
     : undefined;
+
+  const [oldDiffVersion, setOldDiffVersion] = useState(
+    (
+      props.application.orderedFormResults.edges[0].node.versionNumber - 1
+    ).toString()
+  );
+  const [newDiffVersion, setNewDiffVersion] = useState(
+    props.application.orderedFormResults.edges[0].node.versionNumber.toString()
+  );
+  const [showDiff, setShowDiff] = useState(false);
```

Comment:
Can those be stored in the URL query string instead of in the component's internal state?